lib/cn/omf.h

```diff
@@ -226,21 +229,35 @@ OMF_SETGET(struct wbt_lfe_omf, lfe_kmd, 16)
 
 /*****************************************************************
  *
- * Vblock header OMF
+ * Vblock footer OMF
  *
  ****************************************************************/
 
-#define VBLOCK_HDR_MAGIC ((u32)0xea73feed)
+#define VBLOCK_FOOTER_MAGIC ((u32)0xea73feed)
+#define VBLOCK_FOOTER_LEN   4096
```

Comment:
I like to use something I can recognize in a hex dump, like (u32)('v' << 24 | 'b' << 16 | 'f' << 8 | 'm') for magic numbers..

Reply:
Good idea, we can make a separate pass and change all the omf magics in HSE. There's no better time than HSE 3.0 to do this.